acctest/plugin/plugin_acc_test.go

```diff
@@ -69,27 +67,24 @@ func TestAccInitAndBuildBasicAmazonEbs(t *testing.T) {
 	acctest.TestPlugin(t, testCase)
 }
 
-func cleanupPluginInstallation(plugin addrs.Plugin) error {
-	home, err := homedir.Dir()
+func pluginDirectory(plugin addrs.Plugin) (string, error) {
+	pluginDir, err := packer.PluginFolder()
 	if err != nil {
-		return err
+		return "", err
 	}
-	pluginPath := filepath.Join(home,
-		".packer.d",
-		"plugins",
-		plugin.Hostname,
-		plugin.Namespace,
-		plugin.Type)
 
-	if xdgConfigHome := os.Getenv("XDG_CONFIG_HOME"); xdgConfigHome != "" {
-		pluginPath = filepath.Join(xdgConfigHome,
-			"packer",
-			"plugins",
-			plugin.Hostname,
-			plugin.Namespace,
-			plugin.Type)
-	}
+	pluginParts := []string{pluginDir}
+	pluginParts = append(pluginParts, plugin.Parts()...)
+	pluginPath := filepath.Join(pluginParts...)
 
+	return pluginPath, nil
+}
```

Comment:
nit: We can simplify this a little
```suggestion
	return filepath.Join(pluginParts...), nil
}
```